$Id: fix_pagecss.patch 379 2012-09-20 11:45:26Z tg $

Fix PageCSS extension:
* parser hooks are expected to return strings, not void
  => fixes those UNIQ...QINU things showing

--- a/dist/mediawiki-extensions-base/usr/share/mediawiki-extensions/base/PageCSS/PageCSS.php
+++ b/dist/mediawiki-extensions-base/usr/share/mediawiki-extensions/base/PageCSS/PageCSS.php
@@ -38,6 +38,7 @@ class CssHook {
 </style>
 EOT
 		);
+		return "";
 	}
 
 }
